academy/3-ibc/1-what-is-ibc.md

```diff
@@ -141,7 +141,7 @@ IBC clients and transactions assume the trust model of the chains they are conne
 
 All tokens transferred over a particular channel will be assigned the same denomination as other tokens flowing over the channel, but a different one than the same assets between the same chains would have if they were sent across a different channel. The IBC denom looks like `ibc/<hash of the channel-id & port-id>`.
 <br></br>
-You can find more detailed information in the tutorial on [IBC denoms](https://tutorials.cosmos.network/tutorials/understanding-ibc-denoms/#what-is-this-ibc-denom).
+You can find more detailed information in the tutorial on [IBC denoms](https://interchainacademy.cosmos.network/tutorials/5-ibc-dev/).
```

Comment:
Wait, this is the wrong platform! This section is in the academy content, so the link should point to the main academy platform. Lmftfy.